src/shared/bus-polkit.c

```diff
@@ -262,12 +329,12 @@ static int process_polkit_response(
         assert(ret_error);
 
         assert(q->reply);
+        assert(streq(q->action, action));
```

Comment:
Maybe, better to add `assert(q->action)`, or use `streq_ptr()`.

Reply:
Yeah. `q->action` must be set, so it should be checked.